### PR TITLE
TOMEE-2888 Translate to Portuguese: examples/quartz-app

### PR DESCRIPTION
**Pull request by Daniel-Dos**
link to jira : https://issues.apache.org/jira/browse/TOMEE-2888

### CODE REVIEW

File: examples/quartz-app/README_pt.adoc [hunk 1]
```diff
@@ -0,0 +1,257 @@
+:index-group: Java EE Connectors
+:jbake-type: page
+:jbake-status: status=published
+= Uso do Adaptador de Recursos de Quartz
+
+NOTE: Este exemplo é um pouco datado. Ele antecede a API de agendamento que
+foi adicionado ao EJB 3.1. Aplicações modernas devem usar a API de agendamento, que possui muitos, se não todos, os mesmos recursos que o Quartz. De fato, o Quartz é o mecanismo que aciona o suporte `@Schedule` e `ScheduleExpression` no OpenEJB e TomEE.
+
+Apesar de datado de uma perspectiva de programação, ainda é um
```
Comment (hbelmiro) [inline examples/quartz-app/README_pt.adoc:9]:
```suggestion
Apesar de datado de uma perspectiva de programação, ainda é uma
```

File: examples/quartz-app/README_pt.adoc [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+excelente referência sobre como conectar e testar um customizado Adaptador de Recursos do Java EE.
```
Comment (hbelmiro) [inline examples/quartz-app/README_pt.adoc:10]:
```suggestion
excelente referência sobre como conectar e testar um Java EE Resource Adapter personalizado.
```

File: examples/quartz-app/README_pt.adoc [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+quartz-app/quartz-ra/pom.xml
+quartz-app/quartz-ra/src/main/resources/META-INF
+quartz-app/quartz-ra/src/main/resources/META-INF/ra.xml
+----
+
+=== ra.xml
+
+O conector em questão possui adaptadores de recursos de entrada e saída. 
+O adaptador de recursos de entrada pode ser usado para conduzir a mensagem
+beans acionados (MDBs)
```
Comment (hbelmiro) [inline examples/quartz-app/README_pt.adoc:47]:
```suggestion
O adaptador de recursos de entrada pode ser usado para conduzir message driven beans (MDBs)
```

**Comment by Daniel-Dos** [comment]
thanks @hbelmiro  for review.

**Pull request merged**